src/components/SignUp/index.tsx

```diff
@@ -93,8 +94,11 @@ export const SignUpForm = (props: SignUpFormProps) => {
         emailFocused,
         confirmPasswordFocused,
         refIdFocused,
+        resetCaptcha,
     } = props;
 
+    React.useEffect(resetCaptcha, [confirmationError, emailError]);
```

Comment:
- should be tested. Why, because now it called at least once and then after any is changed: `confirmationError`, `emailError`.

So it should have the logic inside:

    useEffect(()=>{ 
        if(confirmationError || emailError){
            resetCaptcha();
        }
    }, [confirmationError, emailError]);